variables.tf

```diff
@@ -46,8 +46,20 @@ variable "create_iam_access_key" {
   default     = true
 }
 
+variable "iam_access_key_max_age" {
+  type        = number
+  description = "Maximum age of IAM access key (seconds). Defaults to 30 days. Set to 0 to disable expiration."
+  default     = 2592000
```

Comment:
Do we want this to be the default going forward ?

Reply:
Yes, @osterman directive has been to make our public modules adhere to best practices by default while giving the consumer the option of opting out.